ocp_resources/network_attachment_definition.py

```diff
@@ -211,8 +213,12 @@ def __init__(
         https://docs.openshift.com/container-platform/4.14/networking/multiple_networks/configuring-additional-network.html#configuration-ovnk-additional-networks_configuring-additional-network
 
         Args:
-            network_name (str): The name of the network. This field is mandatory when not using yaml
-                file.
+            network_name (str): The name of the network, used to connect resources
+                created in different namespaces to the same network. This field is
+                mandatory when not using yaml file.
```

Comment:
not needed `This field is mandatory when not using yaml file`
you don;t have optional in the docstring.

Reply:
ack